integration/actions_test.go

```diff
@@ -88,6 +88,10 @@ var (
 )
 
 func TestApplyPipeline(t *testing.T) {
+	if runtime.GOOS == "windows" {
```

Comment:
Could have a method `TestAllActions` that calls the other action tests?
That way we only need the check on that method

Reply:
I don't think that's a good idea because when any of the actions fail, we will have all the logs together instead of each one its own logs.

Reply:
ok, good to go then!